src/modules/tasklist.js

```diff
@@ -0,0 +1,58 @@
+import Task from './task.js';
+
+export default class TaskList {
+  constructor() {
+    this.localData = JSON.parse(localStorage.getItem('tasklist')) || [];
```

Comment:
- [ ] Data is not being properly saved in the local storage. When data is entered and the page is refreshed, the data vanishes from the UI, and then when a new entry is made the old entries in the local storage are overwritten. Kindly ensure to improve upon this aspect.